packages/web/components/trade-clipboard/index.tsx

```diff
@@ -0,0 +1,481 @@
+    ]);
+
+    const tokenOutSelectCurrencies = useMemo(() => {
+      return tokenSelectCurrencies.filter(
+        (cur) =>
+          cur.coinMinimalDenom !==
+          tradeTokenInConfig.sendCurrency.coinMinimalDenom
+      );
+    }, [
+      tokenSelectCurrencies,
+      tradeTokenInConfig.sendCurrency.coinMinimalDenom,
+    ]);
+
+    const manualSlippageInputRef = useRef<HTMLInputElement | null>(null);
+
+    useEffect(() => {
+      if (isSettingOpen && slippageConfig.isManualSlippage) {
+        // Whenever the setting opened, give a focus to the input if the manual slippage setting mode is on.
+        manualSlippageInputRef.current?.focus();
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [isSettingOpen]);
+
+    return (
+      <div
+        className={classNames(
+          "relative rounded-2xl bg-card border-2 border-cardInner p-2.5",
+          containerClassName
+        )}
+        style={containerStyle}
+        ref={forwardedRef}
+      >
+        <div className="rounded-xl bg-cardInner px-5 pt-5 pb-8">
+          <div className="absolute -top-2 inset-x-1/2 -translate-x-1/2 w-[10rem] h-[3.75rem] z-10 bg-gradients-clip rounded-md">
+            <div className="absolute bottom-0 rounded-b-md w-full h-5 bg-gradients-clipInner" />
+            <div className="absolute inset-x-1/2 -translate-x-1/2 bottom-2 w-12 h-[1.875rem] bg-[rgba(91,83,147,0.12)] rounded-md shadow-[rgba(0,0,0,0.25)_1px_1px_1px_inset]" />
+          </div>
+
+          <div className="relative flex justify-end w-full h-11 mb-[1.125rem]">
+            <button
+              className="relative"
+              onClick={(e) => {
+                e.stopPropagation();
+                setIsSettingOpen(!isSettingOpen);
+              }}
+            >
+              <img
+                className="w-11 h-11"
+                src={`/icons/hexagon-border${
+                  isSettingOpen ? "-selected" : ""
+                }.svg`}
+                alt="hexagon border icon"
+              />
+              <img
+                className="w-5 h-5 absolute inset-1/2 -translate-x-1/2 -translate-y-1/2"
+                src={`/icons/setting${isSettingOpen ? "-selected" : ""}.svg`}
+                alt="setting icon"
+              />
+            </button>
+            {isSettingOpen && (
+              <div
+                className="absolute bottom-[-0.5rem] right-0 translate-y-full bg-card border border-white-faint rounded-2xl p-[1.875rem] z-20 w-full max-w-[23.875rem]"
+                onClick={(e) => e.stopPropagation()}
+              >
+                <div className="subtitle1 text-white-emphasis">
+                  Transaction Settings
+                </div>
+                <div className="flex items-center mt-2.5">
+                  <div className="body2 text-white-disabled mr-2">
+                    Slippage tolerance
+                  </div>
+                  <Tippy
```

Comment:
We do have a `<Tooltip />` component that may serve your needs here.

Reply:
Agreed.